src/built_in/ft_cd.c

```diff
@@ -0,0 +1,44 @@
+#include "../../include/minishell.h"
+
+int go_to_home_directory(t_list *env_list, t_token *token_list, char **path)
```

Comment:
```suggestion
int set_path_to_home_directory(t_list *env_list, t_token *token_list, char **path)
```

함수 동작을 보니  홈디렉토리로 가는 함수가 아니라,   *path를 홈디렉토리로 바꾸는 코드네요. 함수명은 go_~~ 가 아니라 set_~~ 이 적절해보입니다.

Reply:
그게 좋겠군요! 잠시 후에 수정해서 다시 pr 할게요.